.github/workflows/ci.yml

```diff
@@ -72,3 +72,74 @@ jobs:
             cd build && cmake -DBUILD_BPF=On -DUSE_BUNDLED_DEPS=False ../
             KERNELDIR=/lib/modules/$(ls /lib/modules)/build make -j4
             make run-unit-tests
+
+  build-modern-bpf-x86:
+    name: build-modern-bpf-x86 🏗️
+    runs-on: ubuntu-22.04
+    steps:
+
+      - name: Checkout Libs ⤵️
+        uses: actions/checkout@v3
+
+      - name: Install deps ⛓️
+        run: |
+          sudo apt update 
+          sudo apt install -y --no-install-recommends ca-certificates cmake build-essential clang-14 git pkg-config autoconf automake libtool libelf-dev
+
+      - name: Build scap-open using BUNDLED_DEPS 🏗️
+        run: |
+          mkdir -p build
+          cd build && cmake -DUSE_BUNDLED_DEPS=ON -DUSE_MODERN_BPF=ON -DBUILD_LIBSCAP_GVISOR=OFF ../
+          make scap-open
+
+      - name: Check no verifier issues 👀
+        run: |
+          cd build
+          sudo ./libscap/examples/01-open/scap-open --modern_bpf --num_events 0
+
+  test-modern-bpf-x86:
```

Comment:
I'd merge the step with the build-modern-bpf one :) or event better, we could build the modern BPF during the other build jobs (build-libs, build-libs-arm64, build-libs-bundled), then have a single "test-modern-bpf-x86" ?
Wdyt?
Moreover, i think a comment in the workflow saying why we cannot test arm64 would be future-proof :D

Reply:
As fede says, I think we could probably build and test the modern bpf probe with the regular build jobs, it would save an extra build. If you want to keep it as a separate job, we can probably look into using artifacts to share the build results between each other. https://stackoverflow.com/a/57877438

Reply:
Uhm yes, I agree on that, what we could do is merge the 2 `x86 modern probe` in just one job and keep the arm one as it is. Right now I wouldn't merge the modern bpf jobs with the others because they use only the `scap-open` and they don't build all the stuff, moreover, they require the latest image in order to have `bpftool` and `libbpf` latest version. When the probe will be completely supported in `sinsp` we can think of moving it with other jobs maybe in the matrix... WDYT?

Reply:
I agree with @Andreagit97, but could we open an issue to track this? 
I think at some point all those modern ebpf jobs should converge into the others.